docs/installation.rst

```diff
@@ -0,0 +1,69 @@
+
+    .. code-block:: bash
+
+      conda activate root  # Linux / macOS
+      activate root        # Windows
+
+      conda create -n parcels -c conda-forge parcels cartopy
+
+    .. note::
+
+        For some of the examples, ``pytest`` needs to be installed with ``conda install -n parcels pytest``
+
+#. Activate the newly created Parcels environment:
+
+    .. code-block:: bash
+
+      conda activate parcels  # Linux / macOS
+      activate parcels        # Windows</code></pre>
+
+#. Get a zipped copy of the Parcels tutorials and examples from `here <https://docs.oceanparcels.org/en/latest/_downloads/307c382eb1813dc691e8a80d6c0098f7/parcels_tutorials.zip>`_ and unzip it.
+
+#. Go to the unzipped folder and run one of the examples to validate that you have a working Parcels setup:
+
+    .. code-block:: bash
+
+      python example_peninsula.py --fieldset 100 100
+
+    .. note::
+      If you are on macOS and get a compilation error, you may need to accept the Apple xcode license ``xcode-select --install``. If this does not solve the compilation error, you may want to try running ``export CC=gcc``. If the compilation error remains, you may want to check `this solution <https://stackoverflow.com/a/58323411/5172570>`_.
+
+#. Optionally, if you want to run all the examples and tutorials, start Jupyter and open the tutorial notebooks:
+
+    .. code-block:: bash
+
+      jupyter notebook
+
+#. The next time you start a terminal and want to work with Parcels, activate the environment with:
+
+    .. code-block:: bash
+
+      conda activate parcels  # Linux / macOS
+      activate parcels        # Windows
+
+
+Installation for developers
+===========================
+
+Parcels depends on a working Python installation, a netCDF installation, a C compiler, and various Python packages. If you prefer to maintain your own Python installation providing all this, download one of the environment files for `linux <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_linux.yml>`_, for `macOS <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_osx.yml>`_ or for `Windows <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_win.yml>`_ and install it with:
+
+    .. code-block:: bash
+
+      git clone https://github.com/OceanParcels/parcels.git
+      conda env create -f environment_py3_<OS>.yml  # where <OS> is either linux, osx or win
+
+Then, add the directory where you stored the Parcels code to your ``$PYTHONPATH`` environment variable. For example, if you cloned the Parcels code to ``/home/username/parcels``, add the following line to your ``.bashrc`` or ``.zshrc`` file:
+
+    .. code-block:: bash
+
+      export PYTHONPATH=/home/username/parcels:$PYTHONPATH
```

Comment:
Havent extensively tested, `pip install -e /home/username/parcels` seems to work. How does this approach differ? I'm a bit hesitant to directly modify `PYTHONPATH`

Reply:
OK, I now added `pip install -e .` as default option (this was actually suggested too by Github Copilot); but want to keep the `$PYTHONPATH` option as a fall-back (in the past I've had issues with `pip install`)

Reply:
Doing a quick search, I come acrross [this SO article](https://stackoverflow.com/a/49478207) going over developper installation for Conda packages using `conda develop .` ([see help documentation for `conda develop`](https://docs.conda.io/projects/conda-build/en/main/resources/commands/conda-develop.html)).

Has this route been investigated? Happy to do some further investigating if not.

Part of my hesitancy for `$PYTHONPATH` is mainly since I'm not sure how that would affect other Conda environments (i.e. globally "installing" it into all conda/python environments?).

Reply:
OK thanks for digging into this. I can confirm that `pip install --no-build-isolation --no-deps -e .` also seems to work on my system. Have updated it now in the instructions